GoProjects/CalorieCounter/food/food.go

```diff
@@ -1,7 +1,37 @@
 package food
 type Food struct {
-	Name string `json:"name"`
-	Category string `json:"category"`
-	Calories int `json:"calories"`
+	name string `json:"name"`
+	category string `json:"category"`
+	calories int `json:"calories"`
+}
+// setters
+func (f *Food) SetName(name string){
+	f.name = name
+}
+
+func (f *Food) SetCategory(c string){
+	f.category = c
+}
+
+func (f *Food) SetCalories(c int){
+	f.calories = c
+}
+// getters
+func (f *Food) GetName() string {
+	return f.name
+}
+
+func (f *Food) GetCategory() string {
+	return f.category
+}
+
+func (f *Food) GetCalories() int {
+	return f.calories
+}
+
+// builder pattern
+// probably won't need it
```

Comment:
Remove unnecessary comments. This doesnt tell anyone reading the code anything useful